build-tools/packages/build-tools/src/typeValidator/packageJson.ts

```diff
@@ -157,15 +313,18 @@ export type PreviousVersionStyle =
  * @internal
  */
 export async function getAndUpdatePackageDetails(
+    context: Context,
     packageDir: string,
     writeUpdates: boolean | undefined,
-    previousVersionStyle: PreviousVersionStyle,
+    style?: PreviousVersionStyle,
     exactPreviousVersionString?: string,
     resetBroken?: boolean,
 ): Promise<(PackageDetails & { skipReason?: undefined }) | { skipReason: string }> {
     const packageDetails = await getPackageDetails(packageDir);
-
-    if (packageDetails.pkg.name.startsWith("@fluid-internal")) {
+    const pkg = context.fullPackageMap.get(packageDetails.pkg.name);
+    if (pkg === undefined) {
+        return { skipReason: "Skipping package: not found in context" };
```

Comment:
Is there more detail we can give here? I'm not sure I would know what to do with this if I saw it.